bezier/src/triangle.rs

```diff
@@ -215,3 +224,37 @@ fn test_segment_intersection() {
     assert!(!tri.intersects_line_segment(&tri.bc()));
     assert!(!tri.intersects_line_segment(&tri.ac()));
 }
+
+#[cfg(test)]
+fn new_rect(x: f32, y: f32, w: f32, h: f32) -> Rect {
```

Comment:
nit: There is already a rect(x, y, w, h) helper imported in `lyon_bezier/src/lib.rs` doing the same thing.